.azure-pipelines/pr.yml

```diff
@@ -151,3 +151,7 @@ jobs:
   - template: ./windows-steps.yml
     parameters:
       targetBranch: $(System.PullRequest.TargetBranch)
+  - template: ./windows-layout-steps.yml
+    parameters:
+      kind: appx
+      fulltest: true
```

Comment:
This part is just to run the full tests before I merge, to save more post-merge fixups. I'll remove it